.travis.yml

```diff
@@ -84,8 +84,7 @@ before_install:
         contrib/travis_fastfail.sh || exit 1;
         brew tap staticfloat/julia;
         brew rm --force $(brew deps --HEAD julia);
-        brew install -v --only-dependencies --HEAD julia;
-        brew install -v staticfloat/juliadeps/libgfortran;
+        brew install -v gcc gmp mpfr pcre2 cmake staticfloat/julia/openblas-julia staticfloat/julia/arpack-julia staticfloat/julia/suite-sparse-julia staticfloat/juliadeps/libgfortran;
```

Comment:
We don't need `staticfloat/juliadeps/libgfortran` since we're installing `gcc`.  I don't know why that was on the previous line, but we should get rid of it.

Reply:
Done.